app/components/file-menu.js

```diff
@@ -23,6 +23,12 @@ export default Ember.Component.extend({
     share() {
       prompt('Ctrl + C ;-)', window.location.href);
     },
+    embed() {
+      let src = window.location.href.split("?")[0];
+      src += "?numColumns=0";
+      let embedCode = `<iframe width="800" height="600" src="${src}"></iframe>`;
```

Comment:
That's really cool. But we are already using href in Share Twiddle. I'm going to open a PR ASAP and update Ember Twiddle for security reasons.